src/controllers/bulk.js

```diff
@@ -15,6 +23,48 @@ class BulkController extends BaseController {
       .then(response => response.result);
   }
 
+  /**
+   * Create or replace a document directly into the storage engine.
+   * https://docs.kuzzle.io/core/1/api/controllers/bulk/write/
```

Comment:
(nitpicking) use [@link](https://jsdoc.app/tags-inline-link.html):

```suggestion
   * {@link https://docs.kuzzle.io/core/1/api/controllers/bulk/write/|Official documentation}
```